packaging/debian/storagenode.config

```diff
@@ -187,16 +187,16 @@ case "$STATE" in
 	      if [[ ! "$diskspace" =~ ^[0-9]+[GTgt]{1} ]]; then
 		  error_ask_continue storagenode/diskspace/wrong_format
 	      else
-		  space_amount="${diskspace%*[GTgt]}"
-		  if [ "$space_amount" -lt "500" ]; then
+		  input_diskspace_bytes=`numfmt --from=iec $diskspace`
+		  min_diskspace_bytes=`numfmt --from=iec 550G` # minimum diskspace with overhead
+
+		  if [ "$input_diskspace_bytes" -lt "$min_diskspace_bytes" ]; then
 		      error_ask_continue storagenode/diskspace/below_minimum
 		  fi
 		  if [ "$ERR_STATE" != "$ERR_BACK" ]; then
-		      space_unit="${diskspace##*[0-9]}"
-		      freespace=`df --output=avail -B "$space_unit" "$storagedir" |tail -n 1`
-		      freespace_amount="${freespace%*[GT]}"
+		      freespace_bytes=`df --output=avail "$storagedir" |tail -n 1`
```

Comment:
I believe `df` will return its output in 512k blocks still, rather than bytes. You can use `df -B1 --output=avail` to get the actual number of bytes, which can then be compared directly with `$input_diskspace_bytes`.